source/Octopus.Tentacle/Octopus.Tentacle.csproj

```diff
@@ -95,7 +95,7 @@
 		<PackageReference Include="System.Diagnostics.EventLog" Version="4.5.0" />
 		<PackageReference Include="System.Text.Encoding.CodePages" Version="6.0.0" />
 		<PackageReference Include="System.DirectoryServices.AccountManagement" Version="4.5.0" />
-		<PackageReference Include="System.Security.Cryptography.ProtectedData" Version="4.5.0" />
+		<PackageReference Include="System.Security.Cryptography.ProtectedData" Version="5.0.0" />
```

Comment:
This comes from a bump of some nuget packages in Halibut to add the Memory Cache